images/kubekins-e2e/variants.yaml

```diff
@@ -1,32 +1,32 @@
 variants:
   experimental:
     CONFIG: experimental
-    GO_VERSION: 1.12.1
-    K8S_RELEASE: stable
+    GO_VERSION: 1.12.7
+    K8S_RELEASE: latest
     BAZEL_VERSION: 0.28.1
     UPGRADE_DOCKER: 'true'
   master:
     CONFIG: master
-    GO_VERSION: 1.12.1
+    GO_VERSION: 1.12.7
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.23.2
+  '1.16':
+    CONFIG: '1.16'
+    GO_VERSION: 1.12.7
+    K8S_RELEASE: latest-1.16
```

Comment:
historically it's unclear what the latest-but-unreleased branch should be (ref: https://github.com/kubernetes/test-infra/pull/13870/files#r313601983), since it's been `stable` and `latest` in the past, WDYT @BenTheElder ?

```
$ curl https://storage.googleapis.com/kubernetes-release/release/stable.txt
v1.15.2
$ curl https://storage.googleapis.com/kubernetes-release/release/latest.txt
v1.17.0-alpha.0
$ curl https://storage.googleapis.com/kubernetes-release/release/latest-1.16.txt
v1.16.0-beta.0
```

Reply:
I'm fine with `latest-1.16` so long as someone or something remembers to switch over to `stable-1.16` once release has been cut; other alternative is default everything to `latest-1.y` which would require no such post-release toil, but means that unreleased versions of kubectl are the fallback in these images